### ISSUE TITLE
vinyl build system

### ISSUE THREAD

**Issue by CFGrote** [PaNOSC-ViNYL/ViNYL-project#37]
vinyl should expose the following libraries:

- [ ] pyvinyl for the APIs
- [ ] SIMEX 
- [ ] McStass Script
- [ ] McStass
- [ ] jupyter
- [ ] oasys

To be bundled in a docker package and runnable on jupyter hub cloud instance, HPC systems, binder.

I suggest spack (spack.io) do manage the dependencies.

**Comment by ejcjason** [comment]
How should we start it with SimEx? Is the goal for SimEx here the same as this [SIMEX docker task](https://github.com/PaNOSC-ViNYL/ViNYL-project/issues/31)?

**Comment by CFGrote** [comment]
yes, both are related. we need a solution (prefer cmake) that downloads the mentioned codes/libraries and installs them to the user space.
